--- IntroProbability.md	2017-07-07 12:12:27.767107600 +0900
+++ "\347\242\272\347\216\207\350\253\226\345\205\245\351\226\200.md"	2017-07-07 12:12:31.200482800 +0900
@@ -65,12 +65,10 @@
 \int\dotsi\int \rho(x_1,\ldots,x_n)\,dx_1\cdots dx_n = 1
 $$
 を満たしていると仮定する. ただし, 積分領域を省略して書いた場合には $\R^n$ 全域にわたる積分を意味すると約束しておく. このとき期待値汎函数を
-```math
 \begin{align*}
 &E[f(X_1,\ldots,X_n)]\\
 &=\int\dotsi\int f(x_1,\ldots,x_n)\rho(x_1,\ldots,x_n)\,dx_1\cdots dx_n
 \end{align*}
-```
 と定めることができる. このとき, $(X_1,\ldots,X_n)$ は **確率密度函数**  $\rho(x_1,\ldots,x_n)$ を持つと言う.
 
 **例.** 確率変数 $X$ が
@@ -110,14 +108,12 @@
 f(x)\leqq a(x-\mu)+f(\mu)
 $$
 を満たすものが存在する. ゆえに
-```math
 \begin{align*}
 E[f(X)]
 &\leqq E[a(X-\mu)+f(\mu)]\\
 &=a(E[X]-\mu)+f(\mu)
 =f(E[X]).
 \end{align*}
-```
 1つ目の不等号は期待値汎函数の単調性より. 1つ目の等号は期待値汎函数の線形性と規格化条件より. 2つ目の等号は $E[X]=\mu$ より. これで示したい不等式が示された. $f(x)$ が下に凸な場合も同様である. **証明終**
 
 **注意.** $0\leqq t\leqq 1$ のとき
@@ -241,7 +237,6 @@
 \end{cases}
 $$
 となる. ゆえに
-```math
 \begin{align*}
 \sigma_n^2
 &=E[(M_n-\mu)^2]
@@ -249,7 +244,6 @@
 &=\frac1{n^2}\sum_{k,l=1}^nE[X^\prime_k X^\prime_l]
 =\frac{\sigma^2}n.
 \end{align*}
-```
 すなわち, サンプル平均の分散は母集団分散のサンプルサイズ分の1になる. 特にサンプル平均の分散はサンプルサイズが大きくなると小さくなる.
 
 Chebyshevの不等式をサンプル平均 $M_n$ に適用すると, 任意の $a >0$ に対して
@@ -271,7 +265,6 @@
 平均 $0$, 分散 $1$ の正規分布を **標準正規分布** と呼ぶ.
 
 $X$ が標準正規分布に従う確率変数ならば $\mu\in\R$, $\sigma >0$ に対して $Y=\mu+\sigma X$ は平均 $0$, 分散 $\sigma$ の正規分布に従う. なぜならば
-```math
 \begin{align*}
 E[f(Y)]
 &=E[f(\mu+\sigma X)]
@@ -279,13 +272,11 @@
 &=\int f(y)\frac{e^{-(y-\mu)^2/(2\sigma^2)}}{\sqrt{2\pi}}\,\frac{dy}{\sigma}\\
 &=\int f(y)\frac{e^{-(y-\mu)^2/(2\sigma^2)}}{\sqrt{2\pi\sigma^2}}\,dy
 \end{align*}
-```
 3つ目の等号で $x=(y-\mu)/\sigma$ とおいた. これは $Y$ が平均 $\mu$, 分散 $\sigma^2$ の正規分布に従うことを意味する.
 
 一般に確率変数 $X$ の平均が $0$ で分散が $1$ のとき, $Y=\mu+\sigma X$ は平均が $0$ で分散が $\sigma^2$ の確率変数になる.
 
 $X,Y$ は独立な確率変数の組であり, どちらも標準正規分布に従っていると仮定する. このとき任意の $a,b\in\R$, $(a,b)\ne(0,0)$ に対して $Z=aX+bY$ は平均 $0$, 分散 $a^2+b^2$ の標準正規分布に従う確率変数になる. この結果を **正規分布の再生性** という. その証明は以下の通り: $a\ne 0$ と仮定する.
-```math
 \begin{align*}
 E[f(Z)]
 &=E[f(aX+bY)]\\
@@ -294,7 +285,6 @@
 &=\int f(z)\frac{e^{-z^2/(2(a^2+b^2))}}{\sqrt{(2\pi)^2}}\frac{\sqrt{2(a^2+b^2)\pi}}{a^2+b^2}\,dz\\
 &=\int f(z)\frac{e^{-z^2/(2(a^2+b^2))}}{\sqrt{2\pi(a^2+b^2)}}\,dz
 \end{align*}
-```
 3つ目の等号で
 $$
 x = \frac{az-bw}{a^2+b^2}, \qquad y = \frac{bz+aw}{a^2+b^2}
@@ -346,7 +336,6 @@
 $f$ は有限区間の外で $0$ になる $C^3$ 級函数なので, $f'''(x)$ は有限区間の外で $0$ になる連続函数になる. ゆえに $|f'''(x)|/6$ は最大値 $M$ を持ち, 上のTaylor定理から得られた等式の右辺の最後の項の絶対値は $M|h|^3$ 以下になる.
 
 Tayolorの定理から得られる公式を, 独立な確率変数の組 $A,H,K$ で $E[H]=E[K]=0$, $E[H^2]=E[K^2]=1$, $E[|H|^3],E[|K|^3]\leqq C\infty$ で $E[H^3],E[K^3]$ が well-defined なものに適用してみよう.
-```math
 \begin{align*}
 &E\left[f\left(A+\frac{H}{\sqrt n}\right)\right]\\
 &=E[f(A)]+\frac{E[f'(A)]E[H]}{\sqrt n}+\frac{E[f''(A)]E[H^2]}{2n}\\
@@ -354,7 +343,6 @@
 &=E[f(A)]+\frac{E[f''(A)]}{2n}
 +\frac{E[f'''(A+\theta_{A,H/\sqrt n}H/\sqrt n)H^3]}{6n\sqrt n}.
 \end{align*}
-```
 最後の項の絶対値は次のように上からおさえられる:
 $$
 \left|\frac{E[f'''(A+\theta_{A,H/\sqrt n}H/\sqrt n)H^3]}{6n\sqrt n}\right|
@@ -362,21 +350,17 @@
 \frac{MC}{n\sqrt n}.
 $$
 以上をまとめると,
-```math
 \begin{align*}
 &E\left[f\left(A+\frac{H}{\sqrt n}\right)\right]
 =E[f(A)]+\frac{E[f''(A)]}{2n}+R, \\
 &|R|\leqq\frac{MC}{n\sqrt{n}}.
 \end{align*}
-```
 同様の結果が $K$ についても得られるので,
-```math
 \begin{align*}
 &E\left[f\left(A+\frac{K}{\sqrt n}\right)\right]
 =E[f(A)]+\frac{E[f''(A)]}{2n}+S, \\
 &|S|\leqq\frac{MC}{n\sqrt{n}}.
 \end{align*}
-```
 これらの差を取ることによって
 $$
 \left|
@@ -476,13 +460,11 @@
 E[|f(Z_n)-g_m(Z_n)|]\leqq 2M-E[h_m(Z_n)]\to 2M-E[h_m(Y)].
 $$
 ゆえに
-```math
 \begin{align*}
 \limsup_{n\to\infty}E[|f(Z_n)-g_m(Z_n)|]
 &\leqq E[2M-h_m(Y)]\\
 &\leqq \eps + 2M\,P(|Y|>m-1)
 \end{align*}
-```
 
 (2) $g_m(x)$ には上の系が適用できるので,
 $$
